jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -283,12 +284,15 @@ docs_link = env_oss.urls.docs_link;
                 "service_name": this.model.serviceName,
                 "approvers": approversPayload,
                 "domain": this.model.domainName,
-                "description":this.model.serviceDescription
+                "description":this.model.serviceDescription,
             };
 
     if (this.typeOfService == 'api') {
       payload["runtime"] = this.runtime;
       payload["require_internal_access"] = this.vpcSelected;
+      payload["deployment_targets"] = {
```

Comment:
This must be configuration driven. This should be enabled only when apigee component is enabled by Jazz admin.